backend/src/main/java/de/neuefische/koheis/backend/chractercards/CharacterCard.java

```diff
@@ -0,0 +1,17 @@
+package de.neuefische.koheis.backend.chractercards;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document
```

Comment:
You want to declare your mongoDB collection name here.

```suggestion
@Document("cards")
```

Reply:
👍